### PR TITLE
Feature: home button popup functionality

### PR DESCRIPTION
**Pull request by l-ouis**
@arielshif the fonts are acting up not sure how to integrate them properly (the whole of website is in book antiqua instead of gelica rn, not sure where to put link tag

### CODE REVIEW

**Comment by arielshif** [comment]
I'll take a look and try to make the necessary fixes

**Comment by l-ouis** [comment]
Oh i need to add the other buttons so please don't merge yet 😅😅

**Comment by arielshif** [comment]
You pushed your node modules so I wasn't going to. I'm going to check if that's in the gitignore bc it really needs to be

**Comment by l-ouis** [comment]
> You pushed your node modules so I wasn't going to. I'm going to check if that's in the gitignore bc it really needs to be

my bad I accidentally npm install in the wrong folders every other week i'll fix it

File: election-portal/app/assets/home.png
Comment (arielshif) [inline election-portal/app/assets/home.png:1]: will be unnecessary, see later comment

File: election-portal/app/modules/menubar/menu-buttons/home-button.tsx
Comment (arielshif) [inline election-portal/app/modules/menubar/menu-buttons/home-button.tsx:1]: icons should be implemented as react-icons. this icon is MdHome.

ex:
```
import { MdOutlineEmojiEvents, MdOutlineInsertChart, MdHowToVote } from "react-icons/md";
<MdOutlineEmojiEvents className={styles.socialicon} title="Trophy" />
.socialicon {
  vertical-align: middle;
}
```

Comment (arielshif) [inline election-portal/app/modules/menubar/menu-buttons/home-button.tsx:1]: icons should be implemented as react-icons. this icon is MdHome.

ex:
```
import { MdOutlineEmojiEvents, MdOutlineInsertChart, MdHowToVote } from "react-icons/md";
<MdOutlineEmojiEvents className={styles.socialicon} title="Trophy" />
.socialicon {
  vertical-align: middle;
}
```

File: election-portal/app/assets/exit.png
Comment (arielshif) [inline election-portal/app/assets/exit.png:1]: should be MdArrowUpward

**Comment by arielshif** [comment]
things look lovely at first glance but I will provide a more in dept review soon

File: election-portal/app/assets/exit.png
Comment (l-ouis) [reply election-portal/app/assets/exit.png:1]: What about MdExitToApp?
image

If i'm understanding that it's an exit button that would make more sense (If this button serves some other functionality then please lmk)

**Comment by akslaym** [comment]
I am going to go more in depth here soon, but I am errors in menubar.tsx and page.tsx of election-portal-breakdown, that I do ideally want to be resolved before merge. I will comment further here soon

**Comment by l-ouis** [comment]
It doesn't seem like too much work in terms of data parsing was done on main branch, so it'd be nice to get this in before that all happens

Looks like the react-sketch-canvas doesn't work on Firefox, I'll look into it sometime

I should've fixed any errors in menubar.tsx / page.tsx -- lmk if there are any glaring issues. I think a lot of other things should become prettier once we get some sort of centralized context up for the map info. I can work on that once all of this stuff is merged

@arielshif @akslaym

**Review by arielshif — APPROVED** [review]
this is absolutely lovely! great stuff.

I will merge.

@akslaym Based on this, I advise adding the following things as issues to the backlog:

- Add a drop shadow to the drawn lines. This should be 50% opacity, no blur, slight offset to the bottom right
- Changing things to Gelica

**Pull request merged**
